packages/api/package.json

```diff
@@ -9,13 +9,14 @@
   "scripts": {
     "attw": "attw --pack --format table-flipped",
     "build": "tsc",
+    "build:versionedfiles": "NODE_OPTIONS=--no-warnings node --loader ts-node/esm bin/buildVersionedFiles.ts",
```

Comment:
i don't love this file name nor its corresponding `package.json` script name, open to ideas here 😬